content/page/get-involved.md

```diff
@@ -0,0 +1,64 @@
+---
+author: kushinauwu
+cover:
+  alt: obf_membership
+  image: /wp-content/uploads/2019/02/obf_membership.jpg
+date: "2019-02-04T19:04:53+00:00"
+guid: https://www.open-bio.org/wp/?page_id=2546
+title: Get Involved
+url: /get-involved/
+aliases:
+  - /membership/
+
+---
+# Join the OBF community
+
+The OBF is a community that invites anyone who is involved in or plans to become involved in promoting open source or open science in a biological field to get involved.
+There are many ways you can become part of our growing community, regardless of your career state, experience with open source/science.
+
+## Join our online spaces
+
+The OBF community has a number of different online spaces that are open to everyone and are used to network and discuss anything related to open bioinformatics. You can:
+
+* [Subscribe to our regular OBF newsletter](https://mailman.open-bio.org/mailman/listinfo/open-bio-l/) (and if you have news to share, you can [also suggest topics](https://github.com/OBF/newsletter/issues).
```

Comment:
```suggestion
* [Subscribe to our OBF newsletter](https://mailman.open-bio.org/mailman/listinfo/open-bio-l/) (and if you have news to share, you can [also suggest topics](https://github.com/OBF/newsletter/issues).
```

Reply:
it's kind of the opposite of "regular" :-) but not sure if we should say something to reassure that it's infrequent ("roughly quarterly"? "several times a year"?) rather than the annoying DAILY newsletters that some organizations like to send

Reply:
I'd argue the main point here is that there is a newsletter one can subscribe to. We don't need to make a commitment, or note some caveats, to its frequency; after all, the frequency is subject to change depending on volunteer resources.